ledger/block/src/ratify/mod.rs

```diff
@@ -24,11 +24,14 @@ use indexmap::IndexMap;
 type Variant = u8;
 /// A helper type to represent the public balances.
 type PublicBalances<N> = IndexMap<Address<N>, u64>;
+/// A helper type to represent the bonded balances.
+type BondedBalances<N> = IndexMap<Address<N>, u64>;
```

Comment:
```suggestion
type BondedBalances<N> = IndexMap<Address<N>, (Address<N>, u64);
```
Gah! I realized that we should instead expect users to provide a validator address along with a bonded amount. This should now line up with the logic in `credits.aleo`.

Reply:
Makes sense - 74d29bf2c
Can you explain why we don't have to use/set the validator_address in the value: https://github.com/AleoHQ/snarkVM/blob/bond_balances_in_genesis/synthesizer/src/vm/finalize.rs#L775
Even though it is clearly a part of `bond_state`: https://github.com/AleoHQ/snarkVM/blob/testnet3/synthesizer/program/src/resources/credits.aleo#L45